plugins/labhub.py

```diff
@@ -239,7 +247,13 @@ def newcomer_difficulty_level(user, iss):
             2. A newcomer asks for assignment to an issue with difficulty higher
                than low.
             """
-            if self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user):
+            if (self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user) and
```

Comment:
this looks overly complicated, and wrong.

if someone is a maintainer, this `if` clause will be skipped, and then the `elif self.GH3_ORG.is_member(user):` block will be executed, which means this method will return `True`.  which doesnt sound right.  Have I misunderstood somehow?